Lib/lib2to3/pgen2/token.py

```diff
@@ -31,43 +31,48 @@
 EQUAL = 22
 DOT = 23
 PERCENT = 24
-BACKQUOTE = 25
```

Comment:
By deleting BACKQUOTE you make it impossible for the 2to3 tool to parse (some) Python 2 code. I don't think that's an acceptable change. It would probably break python-modernize (which is a 3rd party extension of 2to3).

Reply:
Oh. The CI didn't complain. Does lib2to3 lack a test for that?

Reply:
Likely. It doesn't have very good tests. :-(

Reply:
`BACKQUOTE` is not deleted, is now the 63th token.

Reply:
Got it!